### DISCUSSION TITLE
DCN-V2 Model

### DISCUSSION THREAD

**Discussion by laxmimerit** [RUCAIBox/RecBole#1003] [Q&A] [answered]
Is there any plan to implement DCN-V2 in near future?

**Comment by Sherry-XLL** [comment]
@laxmimerit Hello, we just released RecBole v1.0.0 last month, so there will be no significant update to the master branch in the near future. Normally, we need necessary preparation and testing to develop a new model, and DCN-V2 is not in our current development list for the time being. If you are interested in this model, you are welcome to pull request.

**Comment by Sherry-XLL** [comment, marked as answer]
Hello @laxmimerit!

We have released the [1.1.1 version of RecBole](https://github.com/RUCAIBox/RecBole/releases/tag/v1.1.1) on Oct 5 2022, and DCN-V2 model is supported in our framework now. You can refer to #1418 and [documentation](https://recbole.io/docs/user_guide/model/context/dcnv2.html) for more details.

The suggestions and feedback from open source users are our driving force for continuous updating. We welcome you very much to use our library again and give your valuable comments.

**Comment by laxmimerit** [comment]
thanks a lot.
